tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -728,3 +728,9 @@ test_i2c_lsm6dso16is: lsm6dso16is@6f {
 	reg = <0x6f>;
 	irq-gpios = <&test_gpio 0 0>;
 };
+
+test_tcn75a: tcn75a@6a {
+	compatible = "microchip,tcn75a";
+	reg = <0x6a>;
+	alert-gpios = <&test_gpio 0 0>;
+};
```

Comment:
Update the address to 0x70

Reply:
Looks like another sensor driver merged, so 0x70 is now taken. Address is 0x71 in latest push.